chef_master/source/install_chef_air_gap.rst

```diff
@@ -0,0 +1,602 @@
+
+   .. code-block:: ruby
+
+      app = data_bag_item('apps', 'supermarket')
+
+#. Set the attributes from the data bag:
+
+   .. code-block:: ruby
+
+      node.override['supermarket_omnibus']['chef_server_url'] = app['chef_server_url']
+      node.override['supermarket_omnibus']['chef_oauth2_app_id'] = app['chef_oauth2_app_id']
+      node.override['supermarket_omnibus']['chef_oauth2_secret'] = app['chef_oauth2_secret']
+      node.override['supermarket_omnibus']['package_url'] = app['package_url']
+
+   Note that the ``['package_url']`` setting points to the location of the Supermarket package on your artifact store. When finished, the ``/recipes/default.rb`` file should have code similar to:
+
+   .. code-block:: ruby
+
+      app = data_bag_item('apps', 'supermarket')
+
+      node.override['supermarket_omnibus']['chef_server_url'] = app['chef_server_url']
+      node.override['supermarket_omnibus']['chef_oauth2_app_id'] = app['chef_oauth2_app_id']
+      node.override['supermarket_omnibus']['chef_oauth2_secret'] = app['chef_oauth2_secret']
+
+      include_recipe 'supermarket-omnibus-cookbook'
+
+   Alternatively, if you chose not to use a data bag to store these values, your ``default.rb`` should look similar to this:
+
+   .. code-block:: ruby
+
+      node.override['supermarket_omnibus']['chef_server_url'] = 'https://chef-server.example.com:443'
+      node.override['supermarket_omnibus']['chef_oauth2_app_id'] = '0bad0f2eb04e935718e081fb71asdfec3681c81acb9968a8e1e32451d08b'
+      node.override['supermarket_omnibus']['chef_oauth2_secret'] = '17cf1141cc971a10ce307611beda7ffadstr4f1bc98d9f9ca76b9b127879'
+      node.override['supermarket_omnibus']['package_url'] = 'http://packages.example.com/supermarket_3.1.22-1_amd64.deb'
+
+
+      include_recipe 'supermarket-omnibus-cookbook'
+
+#. Save and close the ``recipes/default.rb`` file.
+
+#. Upload all of your cookbooks to the Chef server:
+
+   .. code-block:: ruby
+
+      knife cookbook upload -a
+
+Bootstrap Supermarket
+-----------------------------------------------------
+Bootstrap the node on which Chef Supermarket is to be installed. For example, to bootstrap a node running Ubuntu on Amazon Web Services (AWS), the command is similar to:
+
+.. code-block:: bash
+
+   $ knife bootstrap ip_address -N supermarket-node -x ubuntu --sudo
+
+where:
+
+* ``-N`` defines the name of the Chef Supermarket node: ``supermarket-node``
+* ``-x`` defines the (ssh) user name: ``ubuntu``
+* ``--sudo`` ensures that sudo is used while running commands on the node during the bootstrap operation
+
+When the bootstrap operation is finished, do the following:
+
+#. Edit the node to add the wrapper cookbook's ``/recipes/default.rb`` recipe to the run-list:
+
+   .. code-block:: bash
+
+      $ knife node edit supermarket-node
```

Comment:
This has issues when run from a windows workstation.  I suggest `knife node run_list set supermarket-node recipe[my_supermarket_wrapper::default]`